src/Project/TrnSite/code/Controllers/BreadcrumbNavController.cs

```diff
@@ -14,6 +14,8 @@ public class BreadcrumbNavController : Controller
         // GET: Breadcrumb
         public ActionResult Index()
         {
+            var siteStartPath = Sitecore.Context.Site.StartPath;
```

Comment:
You would have combines this line with the next line

Reply:
i have implemented the changes kindly review again

Reply:
Thanks merging